core/modules/filter/tests/filter.test

```diff
@@ -1945,10 +1954,10 @@ EOF;
     $f = _filter_htmlcorrector('<p>test\n</p>\n');
     $this->assertEqual($f, '<p>test\n</p>\n', 'HTML corrector -- New-lines are accepted and kept as-is.');
 
-    // cspell:disable
+    // cspell:disable.
```

Comment:
There are must not be periods, after CSpell directives, or CSpell will not recognize them. (I guess it is what happens here, since PHP_CodeSniffer complains about a missing empty line after the comment.)

Reply:
I removed the periods.